### DISCUSSION TITLE
Support for Windows?

### DISCUSSION THREAD

**Discussion by decipherIO** [readysettech/readyset#3] [Q&A] [answered]
Hello,
I am trying to test ReadySet locally to see if it could solve a problem we encountered at my organization. I wanted to know if you have support for deployment on a local Windows machine. I read the README document and tried running the deploy commands. This was what I got:
```
$ bash -c "$(curl -sSL https://launch.readyset.io)"
Sorry, we don't currently have support for your system
Detected platform: x86_64-pc-windows-gnu
```

Do you happen to have support for Windows?

**Comment by alanarenda** [comment, marked as answer]
Hi, we don't have immediate plans to support Windows. We'll follow up here if that changes!
